bindings/python/doc/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../.'))
+
+import sphinx_rtd_theme
+
+# -- Project information -----------------------------------------------------
+
+project = 'libiio'
+copyright = '2020, Analog Devices, Inc.'
+author = 'Paul Cercueil <paul.cercueil@analog.com>'
```

Comment:
curios: do we want to keep using Paul's non-working email?
or is it working now;
i did send some kernel patches using emails, and I got a lot of reply-backs that it wasn't working;

Reply:
Ill remove the email

Reply:
Thats a good point - does it need to get updated in all the fies?

Reply:
@rgetz you mean in the non-python files too?